lib/DataSheet.js

```diff
@@ -149,7 +149,7 @@ var DataSheet = function (_PureComponent) {
         var start = this.state.start;
 
         var pastedMap = [];
-        var pasteData = e.clipboardData.getData('text/plain').split(/\n|\r/).map(function (row) {
+        var pasteData = e.clipboardData.getData('text/plain').split(/\r\n|\n|\r/).map(function (row) {
```

Comment:
this change occurred in my `lib` directory automatically after I installed dependencies. I think the file in master is slightly out of date